db/migrate/20200301015408_add_assocation_between_assignments_and_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddAssocationBetweenAssignmentsAndUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :assignments, :assigned_by_id, :integer
```

Comment:
these should both also be t.references, but to participants for assigned to and staffs for assigned by